RELEASE.md

```diff
@@ -0,0 +1,22 @@
+Release type: patch
+
+This release adds a new exception called `InvalidArgument` and it will be raised when we use Union or Interface as an argument type
+For example this will raise an exception:
+```python
+￼import strawberry
+￼
+@strawberry.type
+class Noun:
+    text: str
+
+@strawberry.type
+class Verb:
+        text: str
+
+Word = strawberry.union("Word", types=(Noun, Verb))
+
+@strawberry.field
+def add_word(word: Word) -> bool:
+    _word = word
+    return True
```

Comment:
```suggestion
For example this will raise an exception:

```python
import strawberry
￼
@strawberry.type
class Noun:
    text: str

@strawberry.type
class Verb:
    text: str

Word = strawberry.union("Word", types=(Noun, Verb))

@strawberry.field
def add_word(word: Word) -> bool:
	...
```

(fixed up some indentation issues)